ToolsTest/Models/VladyslavClassArticleTests/EditorTestProperty_nUnit.cs

```diff
@@ -0,0 +1,51 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+using Tools.Models;
+
+namespace ToolsTest.Models.VladyslavClassArticleTests
+{
+    [TestFixture]
+    public class EditorTestProperty_nUnit
+    {
+        [Test]
+        public void Editor_HasTitle_NotNull ()
+        {
+            //Arrange
+            string title = "Title";
+            //Act
+            Article _editor = new Article(title);
+            //Assert
+            Assert.IsNotNull(_editor.HasTitle);
+        }
+        [Test]
+        public void Editor_HasTitleHasText_NotNullEqual()
+        {
+            //Arrange
+            string title = "Title";
+            string text = "Text";
+            //Act
+            Article _editor = new Article(title,text);
+            //Assert
+            Assert.IsNotNull(_editor.HasTitle);
+            Assert.IsNotNull(_editor.HasText);
+            Assert.That(title, Is.EqualTo(_editor.Title));
+            Assert.That(text, Is.EqualTo(_editor.Text));
+        }
+        [Test]
+        public void Editor_HasText_TypeStringNotNull()
+        {
+            //Arrange
+            string title = "Title";
+            string text = "Text";
+            //Act
+            Article _editor = new Article(title,text);
+            //Assert
+            Assert.IsInstanceOf<string>(_editor.Title);
+            Assert.IsInstanceOf<string>(_editor.Text);
+            Assert.IsNotNull(_editor.HasText);
```

Comment:
1. The properties are strongly typed so do not use checking the type of that properties (`IsInstanceOf<>()`) as there is no any possibility to fail the test here.
2. The `HasText` property returns the boolean value that always returns not-null value. Check that value is true instead.

Also you can use the other ways of setting the text in an article like:
```csharp
//Act
Article _article = new Article(title);
_article.UpdateText(text);
//Assert
Assert.True(_article.HasText);
```
Or to check the situation when this property should return false, like:
```csharp
//Act
Article _article = new Article(title);
//Assert
Assert.False(_article.HasText);
```

```suggestion
            //Act
            Article _article = new Article(title, text);
            //Assert
            Assert.True(_article.HasText);
```